easybuild/easyconfigs/m/minieigen/minieigen-0.5.3_Eigen.patch

```diff
@@ -0,0 +1,24 @@
+diff -ru minieigen-0.5.3.orig/setup.py minieigen-0.5.3/setup.py
```

Comment:
@jrbosch please add a line of comment above this line explaining the patch, and mention yourself as an author:

```
correctly specify location of Eigen include files (provide via EasyBuild)
author: ...
```